Dropdown/Dropdown.js

```diff
@@ -14,53 +14,101 @@
  * @exports DropdownBase
  * @exports DropdownDecorator
  */
-import {on, off} from '@enact/core/dispatcher';
-import {forKey, forward, forProp, handle} from '@enact/core/handle';
-import hoc from '@enact/core/hoc';
-import {add} from '@enact/core/keymap';
+import {forward, forProp, handle} from '@enact/core/handle';
+import EnactPropTypes from '@enact/core/internal/prop-types';
 import kind from '@enact/core/kind';
 import {extractAriaProps} from '@enact/core/util';
-import Spotlight from '@enact/spotlight';
+import {I18nContextDecorator} from '@enact/i18n/I18nDecorator';
 import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
 import Changeable from '@enact/ui/Changeable';
-import Group from '@enact/ui/Group';
+import ForwardRef from '@enact/ui/ForwardRef';
 import IdProvider from '@enact/ui/internal/IdProvider';
-import ri from '@enact/ui/resolution';
 import Toggleable from '@enact/ui/Toggleable';
-import Transition from '@enact/ui/Transition';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import compose from 'ramda/src/compose';
+import not from 'ramda/src/not';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import warning from 'warning';
 
 import Button from '../Button';
+import ContextualPopupDecorator from '../ContextualPopupDecorator';
 import Icon from '../Icon';
 import Item from '../Item';
-import RadioItem from '../RadioItem';
-import Scroller from '../Scroller';
 import Skinnable from '../Skinnable';
 
+import DropdownList from './DropdownList';
```

Comment:
Please move this one line up. Add empty line after it.

Reply:
i just added an empty line after it, according to the import rule 👍